app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt

```diff
@@ -190,4 +192,11 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
             mutable = mutable
         )
     }
+
+    private fun isEmptyMaskBitmap(): Boolean {
+        val maskBitmap = binding.photoMaskView.getMaskBitmap()
+        val emptyBitmap: Bitmap =
+            Bitmap.createBitmap(maskBitmap.width, maskBitmap.height, maskBitmap.config);
```

Comment:
Seems a bit wasteful to create this on every call to the method. You reckon the width/height/config may change between samples?

Reply:
Good point. On the other hand, you probably don't want this large bitmap linger in memory after we are done with the comparison. Ideally, we would have some in-memory LRU cache for bitmaps, which we could also use for other parts where we use (disk) caching. Maybe this would be a good issue to open for #hacktoberfest :D

Reply:
Aight let's call this out-of-scope :)